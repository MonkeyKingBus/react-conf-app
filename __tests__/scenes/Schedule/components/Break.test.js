// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Break from '../../../../app/scenes/Schedule/components/Break';

describe('Schedule - Break', () => {
	it('renders correctly', () => {
		const tree = renderer.create(
			<Break status="present" />
		).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
