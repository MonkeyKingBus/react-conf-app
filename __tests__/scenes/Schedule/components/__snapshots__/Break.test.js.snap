exports[`Schedule - Break renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "stretch",
        "backgroundColor": "white",
        "flexDirection": "row",
      },
      undefined,
    ]
  }>
  <View
    style={
      Object {
        "backgroundColor": "#CCC",
        "width": 6,
      }
    } />
  <BVLinearGradient
    colors={
      Array [
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
        4294967295,
        4294967295,
        4293256677,
        4293256677,
      ]
    }
    end={
      Object {
        "x": 1,
        "y": 0.125,
      }
    }
    locations={
      Array [
        0,
        0.06999999999999999,
        0.06999999999999999,
        0.075,
        0.075,
        0.145,
        0.145,
        0.15,
        0.15,
        0.21999999999999997,
        0.21999999999999997,
        0.22499999999999998,
        0.22499999999999998,
        0.295,
        0.295,
        0.3,
        0.3,
        0.37,
        0.37,
        0.375,
        0.375,
        0.44499999999999995,
        0.44499999999999995,
        0.44999999999999996,
        0.44999999999999996,
        0.52,
        0.52,
        0.525,
        0.525,
        0.595,
        0.595,
        0.6,
        0.6,
        0.6699999999999999,
        0.6699999999999999,
        0.6749999999999999,
        0.6749999999999999,
        0.745,
        0.745,
        0.75,
        0.75,
        0.82,
        0.82,
        0.825,
        0.825,
        0.8949999999999999,
        0.8949999999999999,
        0.8999999999999999,
        0.8999999999999999,
        0.97,
        0.97,
        0.975,
        0.975,
        1.0450000000000002,
        1.0450000000000002,
        1.05,
      ]
    }
    start={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "flexGrow": 1,
        "height": 44,
        "justifyContent": "center",
        "left": 17,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#383838",
            "fontSize": 14,
            "fontWeight": "300",
            "left": -17,
          },
          undefined,
        ]
      }>
       â€” 
      Break
    </Text>
  </BVLinearGradient>
</View>
`;
